impl/src/main/java/org/jboss/weld/logging/BootstrapLogger.java

```diff
@@ -200,4 +202,29 @@
     @LogMessage(level = Level.WARN)
     @Message(id = 147, value = "Decorator {0} declares inappropriate constructor therefore will not available as a managed bean!", format = Format.MESSAGE_FORMAT)
     void decoratorWithNonCdiConstructor(String clazzName);
+
+    @LogMessage(level = Logger.Level.DEBUG)
+    @Message(id = 148, value = "Extension {0} sets annotatedType for {1}", format = Format.MESSAGE_FORMAT)
```

Comment:
What about the following template: `ProcessAnnotatedType.setAnnotatedType() called by extension {0} for {1}`?